### PR TITLE
Update control-panel.md

### PR DESCRIPTION
**Pull request by sbillik**
Hello! I'm a technical writer on the Documentation team at cPanel.

The existing file contains outdated installation instructions for cPanel, as we now prefer the use of WP Toolkit to install WordPress. I've therefore updated the document accordingly and have also combined the relevant information for WP Toolkit for cPanel and Plesk since both control panels use it. My updates have passed review by both a cPanel developer and another technical writer on the cPanel Documentation team.

Note that we would also like to get the page at https://wordpress.org/documentation/article/using-cpanel/ updated accordingly. Thanks in advance for your consideration of these changes!

### CODE REVIEW

**Comment by Crixu** [comment]
Thanks for the work @sbillik. The whole document should be around multiple control panels in the best case and I see that you changed the structure of the document itself. I'll add more details into the inline review

File: server/control-panel.md [hunk 1]
```diff
@@ -1,69 +1,39 @@
 # Control Panels
 
-## cPanel
+This document describes how to install WordPress® in the cPanel & WHM and Plesk control panels using WP Toolkit.
```
Comment (Crixu) [inline server/control-panel.md:3]: This document should be generic for multiple control panels and this sentence should be under the cPanel and Plesk headlines

File: server/control-panel.md [hunk 2]
```diff
@@ -1,69 +1,39 @@
...
 
-This tutorial provides step-by-step examples of creating a database and database user using the cPanel MySQL Database Wizard tool.
+## WP Toolkit
```
Comment (Crixu) [inline server/control-panel.md:5]: Please change back to cPanel or "cPanel WP Toolkit". if you want to cover both Plesk and cPanel, use only one headline like "WP Toolkit for cPanel and Plesk"

File: server/control-panel.md [hunk 3]
```diff
@@ -1,69 +1,39 @@
...
+This document describes how to install WordPress® in the cPanel & WHM and Plesk control panels using WP Toolkit.
 
-This tutorial provides step-by-step examples of creating a database and database user using the cPanel MySQL Database Wizard tool.
+## WP Toolkit 
 
-### Login to your site {#login-to-your-site}
+WP Toolkit is a single management interface that allows you to install, configure, and manage WordPress easily. WP Toolkit can install, configure, and manage WordPress versions 4.9 or later.
 
-Login to your hosting control panel (cPanel) with information provided by your host company.
+cPanel & WHM versions 102 and above install WP Toolkit by default. It is available in Plesk if the server administrator has installed the WP Toolkit extension. The WP Toolkit extension is free with the Web Pro and the Web Host Plesk editions and is available for a fee for the Web Admin edition.
```
Comment (Crixu) [inline server/control-panel.md:9]: please remove any advertising sentences for the panels. If the tool is not available for free it shouldn't be covered in the documentation here.

File: server/control-panel.md [hunk 4]
```diff
@@ -1,69 +1,39 @@
...
 
-### MySQL Database Wizard {#mysql-database-wizard}
+To install WordPress, WP Toolkit retrieves data from wordpress.org. By default, if WP Toolkit cannot establish an initial connection in 15 seconds, it considers wordpress.org unavailable.
```
Comment (Crixu) [inline server/control-panel.md:11]: Please include this in your documentation not here.

**Review by Crixu — CHANGES_REQUESTED** [review]

**Comment by sbillik** [comment]
Thanks for your feedback, @Crixu! I've made the changes you requested. I hope the document now meets WordPress standards!

**Review by Crixu — APPROVED** [review]

**Pull request merged**